### ISSUE TITLE
Reproduce the quantitative evaluation on scannet

### ISSUE THREAD

**Issue by cuirq3** [rakhimovv/npbgpp#6]
Hi! 

Thank you for your great work and clean code! I'm interested in your work and trying to reproduce some results using the ScanNet epoch38 checkpoint given by you.

As far as I understand, the holdout scene for ScanNet is only scene0000_00, so my test result on scene0000_00 should be the same as the paper, which is the performance below:

![image](https://user-images.githubusercontent.com/50020459/195977669-295fa993-2e24-499b-b083-fb844e67721e.png)

However, I can't reach that performance using the checkpoint you released. 

Below is the performance I could reach:

![image](https://user-images.githubusercontent.com/50020459/195977993-63b148e5-5137-40a1-86c8-e4292cb8c496.png)

Here is the command I used to test scene0000_00:

python train_net.py trainer.gpus=1 hydra.run.dir=experiments/<your_test_experiment_folder> datasets=scannet_one_scene datasets.data_root=$\{hydra:runtime.cwd\}/example/scannet datasets.scene_name=scene0000_00 system=npbgpp_sphere system.visibility_scale=0.5 weights_path=./<your_weights_ckpt_file> eval_only=true dataloader=small datasets.n_point=6e6

Did I miss or misunderstand something about the test command?

Thank you in advance!

Best regards!

**Comment by rakhimovv** [comment]
Hi @cuirq3 ! Thanks for your question. These metrics in Table 1 are averaged across holdout scenes (ScanNet-0000, ScanNet-0043, ScanNet-0045). In table 5 in [Supplementary](https://rakhimovv.github.io/npbgpp/files/npbgpp-supp.pdf) you can see metrics for each scene. And for ScanNet-0000 they coincide with what you managed to reproduce.

**Comment by cuirq3** [comment]
Ohhh, I missed the last page of your supplementary!

Thanks a lot, that cleared up my confusion!

And thank you for your quick reply!

**Comment by cuirq3** [comment]
Hi @rakhimovv !

Sorry to bother you again. I'm confused when I'm preprocessing the ScanNet dataset, I followed the sampling method you mentioned in https://github.com/rakhimovv/npbgpp/issues/1#issuecomment-1108333077. But what I got is not the same as the scene0000_00 you released in the example folder.

I used every .sens file of ScanNet. Taking scene0000_00 as an example, I extracted information from scene0000_00.sens - scene0000_02.sens. After sampling every 20th image, I got an images folder with 800+ images. But the example folder you released only contains 279 images.

I guess it might be because you only extract the exact scene00xx_00.sens file and put them in the folder? Or is there another way to build the test examples?

Maybe I missed some key information again, if so, I apologize for my carelessness...

Thank you for your help!

Best regards
